internal/service/alarm_mapper.go

```diff
@@ -222,7 +222,7 @@ func (r *AlarmMapper) MapItem(ctx context.Context,
 	to = data.Object{
 		"alarmEventRecordId": alarmEventRecordId,
 		"resourceID":         resourceID,
-		"resourceTypeID":     resourceTypeID,
+		"resourceTypeId":     resourceTypeId,
```

Comment:
Unfortunately, the AlarmEventRecord is defined using "resourceTypeID" so we need to keep that for this string.

Reply:
OK. Honestly, it isn't easy to understand how the naming convention for the alarmEventRecord was chosen in the standard compared with other attributes.